### DISCUSSION TITLE
Introductions - Please introduce yourself!

### DISCUSSION THREAD

**Discussion by wwahammy** [houdiniproject/houdini#675] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

We've used Zulip in the past but this seems like this might work better so let's give it a try!

**Comment by wwahammy** [comment]
Hey all, I'm Eric Schultz, I'm the main project leader for the Houdini Project. I contribute to Houdini through my work as Lead Developer at CommitChange; we're the main users of Houdini and provide Houdini as a service to nonprofits in the US. I've been involved in free and open source software (FOSS) for over a decade and I'm passionate about making Houdini into a viable, easy to use FOSS tool for processing and managing payments for nonprofits and similar organizations.

In my spare time, I supporter my Milwaukee Bucks and Green Bay Packers and take care of my Yorkie, Penelope. If you spend more than a few minutes with me, you'll probably find me talking about her :smile:

**Comment by wendylbolm** [comment]
Hello, I'm Wendy Bolm, the COO of CommitChange. I pushed for CommitChange to license as open source software early on; now most of what I do with the Houdini project is logging feature requests. 

I have two dogs and two cats and spend most of my free time reading and writing.

**Comment by clarissalimab** [comment]
:wave: Hi ! I'm Clarissa Lima Borges, Brazilian, and I contribute to Houdini through my work as a Junior Developer at CommitChange. My path in FOSS started in my Software Engineering undergrad working on small FOSS projects for the University until I got involved with GNOME through Outreachy and GSoC. When I got to know about Houdini, I identified with the project's mission as someone who helps and has raised money for the animal rescue cause several times using other platforms. I'm happy to make it better for everyone.

In my free time, I am an animal cause activist, Cacau's best friend (who also happens to be my dog), and a casual gamer. I also am a volunteer in an initiative called Big Open Source Sibling, mentoring Brazilian underrepresented folks in tech to introduce them to FOSS.

**Comment by eduardomr** [comment]
👋 Hi everyone!

My name is Eduardo Maia, I'm Brazilian, and I'm currently a software engineering student at the University of Brasília (UnB), where I'm enrolled in the GCES course - Software Configuration and Management. Through this course, my team and I were encouraged to contribute to the Houdini project and we are excited to take on this challenge.

In my spare time, I like to play guitar and play games on PC. I think these activities help me relax and provide a creative outlet outside of my studies.

I look forward to collaborating with you on this project!

**Comment by klyssmannoliveira** [comment]
Hello, everyone! My name is Klyssmann, and I'm a software engineering student currently taking classes in software configuration and management. We were encouraged to join the open-source community, and we chose this repository as our starting point.

I previously studied mechanical engineering (or almost that), so if you have any questions about mechanics, I would be happy to help!

However, I am eager to improve my software skills, so I am always open to learning new things.

**Comment by joseluis-rt** [comment]
👋 Hey everyone!

My name is José Luís, I'm 22 years old and I'm a Brazilian software engineering student at the University of Brasília (UnB), currently taking the GCES course. My group and I are excited to contribute to the Houdini project and learn from the community.

In my free time, I enjoy spending time with my pug named Loki and following the Premier League, where I support Manchester City.

Looking forward to collaborating with you all!

**Comment by lramon2001** [comment]
Hello, everyone!

My name is Lucas Ramon, and I'm a software engineering student at the University of Brasília (UnB). Currently, I'm enrolled in the GCES course - Software Configuration and Management, like Eduardo I'm excited to join the Houdini project with all of you.

In my spare time, I find great joy in riding my motorcycle and exploring new places. There's something liberating about the open road and the opportunity to discover unfamiliar territories. It's a hobby that allows me to relax and appreciate the beauty of the world around me.

I'm really looking forward to collaborating with each of you on this project. I believe that our collective skills and expertise will enable us to accomplish great things together. Let's make the most of this opportunity and create something amazing!

**Comment by Abraao1231** [comment]
Hello, I'm Abraão Alves, a Software Engineering student at the University of Brasília (UnB). I'm currently taking the GCES (Software Configuration and Evolution Management) course, which has sparked my interest in effective software project management and working with open source.

In my free time, I enjoy playing Valorant, a game that helps me relieve the stress of studying, sometimes.

I'm excited to participate in the discussions in this repository because I believe I can contribute with my knowledge in Software Engineering.

**Comment by victorear05** [comment]
Hello, I'm VIctor Eduardo, a Software Engineering student at the University of Brasília (UnB). Like my colleagues currently, I'm enrolled in the GCES course - Software Configuration and Management, and I'm excited join the Houdini project with all of you too.

In my free time, I enjoy playing League Of Legends or other game with friends.

I'm excited to contribute to participate and colaborate with this project.

**Please I'm sorry for trying taking a issue before introducing myself 😅
